app/utils/generic.py

```diff
@@ -216,7 +216,7 @@ def platform_specific_open(path: str | Path) -> None:
     path = str(path)
     if sys.platform == "darwin":
         logger.info(f"Opening {path} with subprocess open on MacOS")
-        if p.is_file() or (p.is_dir() and p.suffix == ".app"):
+        if p.is_dir() and p.suffix == ".app":
```

Comment:
It does change the behavior from what is was prior, but I believe what it was prior is not ideal. On Mac (and in this context), `-R` makes it so that the directory is opened (or, if it is a file, the parent directory). This "special" case is only really necessary for `.app`s, where going without `-R` would try to run the application.

Normal directory and file paths are handled just fine with `open` without `-R`.